packages/contentstack-export/package.json

```diff
@@ -1,14 +1,15 @@
 {
   "name": "@contentstack/cli-cm-export",
   "description": "Contentstack CLI plugin to export content from stack",
-  "version": "1.3.0",
+  "version": "1.4.0",
   "author": "Contentstack",
   "bugs": "https://github.com/contentstack/cli/issues",
   "dependencies": {
     "@contentstack/cli-command": "^1.1.0",
     "@contentstack/cli-utilities": "^1.1.0",
```

Comment:
@aman19K cli-utilities is at it's latest verion 1.2.0, why you are still using 1.1.0?

Reply:
@netrajpatel version updated.